logstash-core/lib/logstash/logging/logger.rb

```diff
@@ -0,0 +1,72 @@
+require "logstash/java_integration"
+
+module LogStash
+  module Logging
+    class Logger
+      java_import org.apache.logging.log4j.LogManager
+      java_import org.apache.logging.log4j.Logger
+      java_import org.apache.logging.log4j.Level
+      java_import org.apache.logging.log4j.core.config.LoggerConfig
+
+      def initialize(name)
+        @logger = LogManager.getLogger(name)
+      end
+
+      def debug?
+        @logger.is_debug_enabled
+      end
+
+      def info?
+        @logger.is_info_enabled
+      end
+
+      def error?
+        @logger.is_error_enabled
+      end
+
+      def warn?
+        @logger.is_warn_enabled
+      end
+
+      def fatal?
+        @logger.is_fatal_enabled
+      end
+
+      def debug(message, data = {})
+        @logger.debug(message, data)
+      end
+
+      def warn(message, data = {})
+        @logger.warn(message, data)
+      end
+
+      def info(message, data = {})
+        @logger.info(message, data)
+      end
+
+      def error(message, data = {})
+        @logger.error(message, data)
+      end
+
+      def fatal(message, data = {})
+        @logger.fatal(message, data)
+      end
```

Comment:
I remember that you mentioned that we could have `trace` but I don't see anything related to log4j2 traces method. If we don't support all the flow tracing we shouldn't mention that option in the settings?